scripts/development/m3_stack/m3coordinator.yml

```diff
@@ -45,3 +45,18 @@ clusters:
         jitter: true
       backgroundHealthCheckFailLimit: 4
       backgroundHealthCheckFailThrottleFactor: 0.5
+
+ingest:
+  ingester:
+    workerPoolSize: 100
+    opPool:
+      size: 100
```

Comment:
Probably by default want to use a reasonable set of defaults that would work for production, i.e. 10000 or something for these values?